src/runtime/pkg/katautils/config.go

```diff
@@ -1140,6 +1143,14 @@ func newDragonballHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 	}, nil
 }
 
+func newRemoteHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
+
+	return vc.HypervisorConfig{
+		RemoteHypervisorSocket:  h.RemoteHypervisorSocket,
```

Comment:
Small detail (a.k.a nit), other hypervisors define the default binary path in `src/runtime/pkg/katautils/config-settings.go` that is returned in case  of `h.path` is empty.  In our case, `/run/peerpod/hypervisor.sock` should be the default socket value, right?

See, for example, the following config initializer:
```
func newFirecrackerHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
	hypervisor, err := h.path()
	if err != nil {
		return vc.HypervisorConfig{}, err
	}
...
...
```
Also should we initialize the timeout with a default value below?

Reply:
I think I've understood the suggestion correctly, so added https://github.com/kata-containers/kata-containers/commit/5ee390461698bc1f3405b98b033d1e193a75ba19, so let me know if that works and I'll fix it up

Reply:
hi @stevenhorsman , let's leave https://github.com/kata-containers/kata-containers/commit/5ee390461698bc1f3405b98b033d1e193a75ba19 for a follow up PR. Is it okay for you?

Reply:
I think I already fixup'd it after I didn't hear anything I'm afraid